adapters/handlers/rest/middlewares.go

```diff
@@ -87,15 +88,17 @@ func makeSetupGlobalMiddleware(appState *state.State) func(http.Handler) http.Ha
 	return func(handler http.Handler) http.Handler {
 		handleCORS := cors.New(cors.Options{
 			OptionsPassthrough: true,
-			AllowedMethods:     []string{"POST", "PUT", "DELETE", "GET", "PATCH"},
+			AllowedMethods:     strings.Split(appState.ServerConfig.Config.CORS.AllowMethods, ","),
+			AllowedHeaders:     strings.Split(appState.ServerConfig.Config.CORS.AllowHeaders, ","),
+			AllowedOrigins:     strings.Split(appState.ServerConfig.Config.CORS.AllowOrigin, ","),
```

Comment:
This `cors.Options` field is called `AllowedOrigins`, which implies multiple origins. But elsewhere in this patch, we see `AllowOrigin`.

If `AllowOrigin` supports multiple origins, would it make sense to rename to `AllowOrigins`?

Reply:
I named it `Origin` to be consistent with what the HTTP header is called: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin